index.jsx

```diff
@@ -269,6 +272,15 @@ var App = React.createClass({
         this.setState({})
     },
 
+    onCellClick: function(cell){
+        if (cell === null){
+            this.selectedCell = [];
+            return;
+        }
+
+        this.selectedCell = [cell];
```

Comment:
selectedCell should be an individual item, should it not?  If it is expected to be an array, i would think the property should be named selectedCells